jsk_arc2017_baxter/euslisp/lib/arc-interface.l

```diff
@@ -344,42 +366,21 @@
       (setq obj-box-z-length (z-of-cube (send self :bbox->cube obj-box)))
       (setq obj-pos (v+ obj-pos (float-vector 0 0 (/ obj-box-z-length 2))))
       obj-pos))
-  (:try-to-pick-object
+  (:try-to-pick-object-v4
```

Comment:
I think :`:try-to-pick-object` is common method for both arm and you should not remove it.
Also it should be meta  method for `:try-to-pinch-object` and `try-to-suction-object`.
try to be like 
`pick-object-in-bin` or `pick-object-in-tote` 
-> `pick-object-with-movable-region`
-> `:try-to-pick-object`
-> `try-to-suction-object` or `try-to-pinch-object`